app/templates/descriptor/confirm_resources_req.html

```diff
@@ -0,0 +1,110 @@
+{% extends 'layouts/base.html' %}
+{% import 'macros/form_macros.html' as f %}
+
+{% block content %}
+    <div class="ui stackable grid container">
+        <div class="sixteen wide tablet twelve wide computer centered column">
+            <h2 class="ui header">
+                {{ desc.name }}
+                <div class="sub header">
+                    Resolve issues with current resources.
+                </div>
+            </h2>
+
+            <a class="ui basic compact button" href="{{ url_for('descriptor.change_option_values_index', desc_id=desc.id) }}">
+                <i class="caret left icon"></i>
+                Back to descriptor
+            </a>
+
+            <h3 class="ui red block header">
+                <i class="warning circle icon"></i>
+                <div class="content">
+                    Action Required
+                    <div class="sub header">
+                        Deleting the value <b> {{ desc.values[option_index] }} </b> from the <b> <u>required</u> descriptor {{ desc.name }} </b>
+                        will affect these resources. Please select the new value for the resources below.
```

Comment:
change the text from "please select the new value for the resources below" (because we don't have the re-assign feature in this page) to "they will no longer have the {{option}} descriptor."